pkgs/development/python-modules/eth-utils/default.nix

```diff
@@ -29,12 +30,18 @@ buildPythonPackage rec {
   ] ++ lib.optional (!isPyPy) cytoolz
   ++ lib.optional isPyPy toolz;
 
-
   nativeCheckInputs = [
+    mypy
```

Comment:
Can we avoid mypy? we don't want to run linters in checks

Reply:
done